cmd/edit/cluster/cmd.go

```diff
@@ -207,6 +210,14 @@ func init() {
 			"followed by a CIDR. \nExample: '--ovn-internal-subnets=\"join=192.168.255.0/24,transit=192.168.255.0/24,"+
 			"masquerade=192.168.255.0/24\"'",
 	)
+
+	flags.StringVar(
+		&args.channelGroup,
+		"channel-group",
+		ocm.DefaultChannelGroup,
+		"Changes the channel group used for cluster versions. "+
+			"Channel group is the name of the channel where this image belongs, for example \"stable\" or \"fast\".",
```

Comment:
Maybe this should capture stable or eus instead, since fast is behind a capability

Reply:
I will remove `fast`